packages/shared/src/components/fields/ProgressBar.tsx

```diff
@@ -35,8 +36,9 @@ export function ProgressBar({
     >
       <meter
         className={classNames(
-          'appearance-none bg-accent-cabbage-default transition-[width]',
+          'appearance-none transition-[width]',
           className?.bar,
+          className?.barColor ?? 'bg-accent-cabbage-default',
```

Comment:
we should really really consider that tailwind merge, this wouldn't be necessary

Reply:
This actually fixes it as a neater solution I think.
If the background is variable (which it is) it should not be defined on the top level. (or should be a prop)

TW merge is technically just another variant of !important but on top level.
Not bad, but think where possible you should still avoid it if you can abstract things away.

Reply:
If we're accepting just a set of colors, this or that or that, then yes a prop makes more sense. 
How it's done at the moment where every possible color set as a classname, I think `className?.bar` would have been more then enough with tailwind merge or !important.